virttest/utils_misc.py

```diff
@@ -2716,25 +2716,37 @@ def set_winutils_letter(session, cmd, label="WIN_UTILS"):
     return cmd
 
 
-def list_linux_guest_disks(session, partition=False):
+def get_linux_disks(session, parted=True):
     """
     List all disks OR disks with no partition in linux guest.
 
     :param session: session object to guest
-    :param partition: if true, list all disks; otherwise,
-                      list only disks with no partition.
+    :param partition: if true, list all disks include partition;
```

Comment:
param parted instead of partition, and the same for the params in function  get_linux_disks_did()/format_linux_disk()/format_guest_disk()